dts/bindings/clock/renesas,smartbond-clock.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas Smartbond clock control node
+
+compatible: "renesas,smartbond-clock"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: false
```

Comment:
remove `required: false` its the default.

Reply:
Done